commons/ava_networks/high_level_gecko_client.go

```diff
@@ -40,6 +43,46 @@ func NewGeckoUser(username string, password string) *GeckoUser {
 }
 
 
+func (highLevelGeckoClient HighLevelGeckoClient) AddValidatorOnSubnet(
+	nodeId string,
+	pchainAddress string,
+	stakeAmount int64) error {
```

Comment:
Can we make sure that when we do a continuation like this, it doesn't line up with the code below? Toss in an extra indent or two? (I know this is Goland's fault)